Annex60/Fluid/HeatPumps/Carnot_TCon.mo

```diff
@@ -160,6 +155,24 @@ Annex60.Fluid.Chillers.Examples.Carnot_TEva</a>.
 revisions="<html>
 <ul>
 <li>
+January 3, 2017, by Michael Wetter:<br/>
+Removed parameters
+<code>effInpEva</code> and <code>effInpCon</code>
+and updated documentation.
+This is for
+<a href=\"https://github.com/iea-annex60/modelica-annex60/issues/497\">
+issue 497</a>.
+</li>
+<li>
+August 8, 2016, by Michael Wetter:<br/>
```

Comment:
I am a bit confused whether this date is intentional chosen or if you forgot to edit it?